modules/model_package/spleen_segmentation/configs/inference.json

```diff
@@ -0,0 +1,142 @@
+{
+    "multi_gpu": false,
+    "amp": true,
+    "model": "#monai.data.load_net_with_metadata('../models/model.ts')[0]",
+    "network": {
+        "name": "UNet",
+        "args": {
+            "spatial_dims": 3,
+            "in_channels": 1,
+            "out_channels": 2,
+            "channels": [16, 32, 64, 128, 256],
+            "strides": [2, 2, 2, 2],
+            "num_res_units": 2,
+            "norm": "batch"
+        }
+    },
+    "preprocessing": [
```

Comment:
I commented below on defining the input and output formats more verbosely so that a transform sequence like this can be inferred with less JSON. This kind of definition is still needed for cases where that isn't going to have enough specific information, so we should establish the JSON format to use for any given transform. For transforms like "Spacingd" it's clear how to provide values in JSON with Python equivalents for each constructor argument, but more complex transforms may need more help. In general though we can represent instantiating a transform using this JSON syntax where a mapping `{"name": "Foo", "args":{...}}` is transformed into a constructor call for class `Foo`.

Reply:
@ericspod thanks for the idea, and about the instantiation logic, I am already developing it in draft PR: https://github.com/Project-MONAI/MONAI/pull/3593/files#diff-7041497533453f2514ad444d001950097e020fce4b9c2e5afee00edb99707d64R72.

Thanks.

Reply:
Hi @ericspod ,

Updated the PR to refer to the `metadata` configs.
More details about how to parse the configs is in this draft PR: https://github.com/Project-MONAI/MONAI/pull/3593.

Thanks for your suggestion!